algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -0,0 +1,253 @@
+import contextlib
+import itertools
+import math
+from typing import Dict, Optional, Tuple
+
+from absl import logging
+import jax
+import torch
+import torch.distributed as dist
+from torch.nn.parallel import DistributedDataParallel as DDP
+
+from algorithmic_efficiency import param_utils
+from algorithmic_efficiency import spec
+from algorithmic_efficiency.pytorch_utils import pytorch_setup
+import algorithmic_efficiency.random_utils as prng
+from algorithmic_efficiency.workloads.librispeech_conformer import metrics
+from algorithmic_efficiency.workloads.librispeech_conformer import workload
+
+from .model import ConformerConfig
+from .model import ConformerEncoderDecoder
+from .model import initialize
+
+USE_PYTORCH_DDP, RANK, DEVICE, N_GPUS = pytorch_setup()
+
+
+class LibriSpeechConformerWorkload(workload.BaseLibrispeechWorkload):
+
+  def init_model_fn(self, rng: spec.RandomState) -> spec.ModelInitState:
+    torch.random.manual_seed(rng[0])
+    model = ConformerEncoderDecoder(ConformerConfig())
+    self._model = model
+    self.ctc_loss = torch.nn.CTCLoss(blank=0, reduction="none")
+    # Run model once to initialize lazy layers
+    t = 320000
```

Comment:
given that this is used below as well, I would pull this into a file level constant like `MAX_INPUT_LENGTH = 320000` and replace all uses of it

Reply:
Done.